test/cookbooks/verify-octo/recipes/tentacle.rb

```diff
@@ -20,9 +20,19 @@
 
 tentacle = node['verify-octo']['tentacle']
 
+# This section will mock out the certificate creation
+directory 'C:\Octopus' do
```

Comment:
This should be handled by the tentacle resource but we need to generate the cert first or it will fail as a remote user